package.json

```diff
@@ -92,6 +92,7 @@
     "babel-plugin-inline-react-svg": "^1",
     "babel-plugin-prismjs": "^2",
     "babel-preset-react-app": "^10",
+    "chromatic": "^5",
```

Comment:
Why is this needed?

Reply:
It's a module for publishing storybook online (see Error component in the first comment). While it's possible to self host storybooks many people don't have their own server, so I used and added chromatic.